tests/OSPSuite.Core.IntegrationTests/Services/QuantityValuesUpdaterIntegrationTests.cs

```diff
@@ -0,0 +1,180 @@
+         _modelConfiguration = new ModelConfiguration(_model, _simulationConfiguration, _simulationBuilder);
+      }
+
+      protected override void Because()
+      {
+         sut.UpdateQuantitiesValues(_modelConfiguration);
+      }
+
+      [Observation]
+      public void should_overwrite_the_parameter_value_with_mean_value_of_the_parameter()
+      {
+         var parameter = _model.Root.EntityAt<IParameter>("Liver", "Param");
+         parameter.Value.ShouldBeEqualTo(20);
+      }
+   }
+
+   internal class When_updating_the_value_of_a_parameter_value_defined_as_formula_using_a_normal_distributed_parameter : concern_for_QuantityValuesUpdaterIntegration
+   {
+      private ModelConfiguration _modelConfiguration;
+      private ModuleConfiguration _moduleConfiguration;
+      private ParameterValue _meanParameterValue;
+      private ParameterValue _distributedParameterValue;
+      private ParameterValue _deviationParameterValue;
+      private ParameterValue _percentileParameter;
+
+      protected override void Context()
+      {
+         base.Context();
+
+         _distributedParameterValue = new ParameterValue
+         {
+            DistributionType = DistributionType.Normal,
+            Name = "Param",
+            ContainerPath = new ObjectPath(_organism.Name, _liver.Name),
+         };
+
+         _meanParameterValue = new ParameterValue
+         {
+            Name = MEAN,
+            ContainerPath = new ObjectPath(_organism.Name, _liver.Name, _parameter.Name),
+            Value = 20
+         };
+
+         _deviationParameterValue = new ParameterValue
+         {
+            Name = DEVIATION,
+            ContainerPath = new ObjectPath(_organism.Name, _liver.Name, _parameter.Name),
+            Value = 5
+         };
+
+         _percentileParameter = new ParameterValue
+         {
+            Name = PERCENTILE,
+            ContainerPath = new ObjectPath(_organism.Name, _liver.Name, _parameter.Name),
+            Value = 0.8
+         };
+
+         _parameterValues.Add(_distributedParameterValue);
+         _parameterValues.Add(_meanParameterValue);
+         _parameterValues.Add(_deviationParameterValue);
+         _parameterValues.Add(_percentileParameter);
+
+         _moduleConfiguration = new ModuleConfiguration(_module, null, _parameterValues);
+         _simulationConfiguration.AddModuleConfiguration(_moduleConfiguration);
+         _simulationBuilder = new SimulationBuilder(_simulationConfiguration);
+         _modelConfiguration = new ModelConfiguration(_model, _simulationConfiguration, _simulationBuilder);
+      }
+
+      protected override void Because()
+      {
+         sut.UpdateQuantitiesValues(_modelConfiguration);
+      }
+
+      [Observation]
+      public void should_overwrite_the_parameter_value_with_the_value_of_the_parameter()
+      {
+         var parameter = _model.Root.EntityAt<IParameter>("Liver", "Param");
+         parameter.Value.ShouldBeEqualTo(24.28, 1e-2);
```

Comment:
value of the distribution using the percentile of 0.8 (assuming this is right, I did not verify...)